tests/Microsoft.Maui.IntegrationTests/TemplateTests.cs

```diff
@@ -0,0 +1,96 @@
+
+namespace Microsoft.Maui.IntegrationTests
+{
+	public class TemplateTests : BaseBuildTest
+	{
+		[Test]
+		// Parameters: short name, target framework, build config, use pack target
+		[TestCase("maui", "net6.0", "Debug", false)]
+		[TestCase("maui", "net6.0", "Release", false)]
+		[TestCase("maui", "net7.0", "Debug", false)]
+		[TestCase("maui", "net7.0", "Release", false)]
+		[TestCase("maui-blazor", "net6.0", "Debug", false)]
+		[TestCase("maui-blazor", "net6.0", "Release", false)]
+		[TestCase("maui-blazor", "net7.0", "Debug", false)]
+		[TestCase("maui-blazor", "net7.0", "Release", false)]
+		[TestCase("mauilib", "net6.0", "Debug", true)]
+		[TestCase("mauilib", "net6.0", "Release", true)]
+		[TestCase("mauilib", "net7.0", "Debug", true)]
+		[TestCase("mauilib", "net7.0", "Release", true)]
+		public void Build(string id, string framework, string config, bool shouldPack)
```

Comment:
Overall I really like that these are Xunit now and you should be able to debug them locally!

Reply:
Yeah being able to run individual test permutations directly in the IDE should help a lot with identifying and diagnosing failures in the future. I think we can move the "Build .NET MAUI Samples" tests out of cake and into this project in a future PR as well.